### DISCUSSION TITLE
Plugin to Share: From Template

### DISCUSSION THREAD

**Discussion by FelipeRearden** [Taitava/obsidian-shellcommands#126] [Offtopic]
Hello @Taitava !!!!

I found this plugin that create forms to insert content inside the notes. It’s called From Template 

https://github.com/mo-seph/obsidian-note-from-template

Might be a good tool for your Vault related to your cat 🐱 . I saw this and remembered of you :)

Have a great day!

**Comment by Taitava** [comment]
Thanks, it looks nice! 😎

And it introduced a new library to me: [mustache.js](https://github.com/janl/mustache.js) that can be used to render `{{variables}}`. I need to check if it can be a useful replacement for SC's current variable system that I've coded myself.

**Reply by FelipeRearden** [reply]
Good to know that you liked and I bring to you a new library!!!!!!
